opsdroid/events.py

```diff
@@ -357,3 +357,60 @@ class Image(File):
         """Return the image dimensions `(w,h)`."""
         fbytes = await self.get_file_bytes()
         return get_image_size_from_bytesio(io.BytesIO(fbytes), len(fbytes))
+
+
+class NewRoom(Event):
+    """Event class to represent the creation of a new room."""
+
+    def __init__(self, name=None, params=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = name
+        self.room_params = params or {}
+
+
+class RoomName(Event):
```

Comment:
all these events need docstrings.

Reply:
These new events are interesting because it's not clear whether they are inbound, outbound or both. Does this event happen when the room name changes? Or do you send it in order to change the room name?

Should events for both of those things exist, or should it just be one event? Is this that one event?

Ad you say docstrings would help a lot here.

Reply:
All these events are designed to be bi-directional, they can be emitted or consumed by connectors. This fact has caused some confusion (@SolarDrew and I got caught out multiple times).